CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.5.0 (2020-04-22)
+
+- Aded the ability to set the county field component's form value via an iso2 country code. [#82](https://github.com/blackbaud/skyux-lookup/pull/82)
```

Comment:
Added is misspelled. Also, we should capitalize ISO. Did a quick search and it looks like it should be have a space between ISO and 2.

Reply:
Hmmm. The property they set is `iso2` maybe we should just say "`iso2` code"? Thoughts?

Reply:
Ah, didn't realize it was a specific property. I think the original works then so long as we place iso2 within ticks.

Reply:
Ok. Done and I fixed the typo